proto/xds.proto

```diff
@@ -1,3 +1,17 @@
+// Copyright Istio Authors
```

Comment:
This is not copyright Istio Authors, its copy+pasted from Envoy

Reply:
Sure, let me fix it.

Reply:
I have reverted those changes, not sure what is the right repo and path they from, so keep it as before.

Maybe we can introduce a new target like `update-xds` in another PR.

Reply:
We don't want to update, it's pruned by hand to remove unused fields to remove bloat